alica_common_config/CMakeLists.txt

```diff
@@ -1,41 +0,0 @@
-cmake_minimum_required(VERSION 3.5.1)
-project(alica_common_config)
-
-add_compile_options(-std=c++14)
-
-# Set up utils to allow Colcon/catkin to find the packages
-include(GNUInstallDirs)
-include(CMakePackageConfigHelpers)
-set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/${CMAKE_PROJECT_NAME}/cmake)
```

Comment:
How is the cflags file that was in this package being used now?

Reply:
I'm more wondering what causes it to actually be used

Reply:
It should be detected by the cmake config helpers, but it looks like it's only working for this package and not for all of them. Looking at a fix.

Reply:
Looks like there was a bug in how we were doing it previously - not all the packages had the compilation flags enabled. That's been fixed by turning the cflags into a macro and calling it in all the packages